pkg/network/image-pull-policy.go

```diff
@@ -21,14 +21,16 @@ func validateImagePullPolicy(conf *opv1alpha1.NetworkAddonsConfigSpec) []error {
 	return []error{}
 }
 
-func fillDefaultsImagePullPolicy(conf, previous *opv1alpha1.NetworkAddonsConfigSpec) {
+func fillDefaultsImagePullPolicy(conf, previous *opv1alpha1.NetworkAddonsConfigSpec) error {
```

Comment:
We should use []error instead like we do in validate functions. So in case multiple components have invalid config, we report all at once.